__tests__/Unit/pages/Standup/Standup.test.tsx

```diff
@@ -0,0 +1,148 @@
+                >)
+        );
+        jest.spyOn(AuthenticatedHooks, 'default').mockImplementation(() => ({
+            isLoggedIn: true,
+            isLoading: true,
+            userData: {
+                userName: 'Pratiyush',
+                firstName: 'Pratiyush',
+                profilePicture: '',
+            },
+        }));
+        const { getByText } = renderWithRouter(
+            <Provider store={store()}>
+                <StandUp />
+            </Provider>,
+            {
+                query: { dev: 'true' },
+            }
+        );
+        expect(getByText('Loading...')).toBeInTheDocument();
+        expect(getByText('Standup')).toBeInTheDocument();
+    });
+
+    test('should not be able to render standup component if user is not logged-in and not authenticated', () => {
+        jest.spyOn(useGetUserQueryHook, 'useGetUserQuery').mockImplementation(
+            (): ReturnType<typeof useGetUserQueryHook.useGetUserQuery> =>
+                ({
+                    isLoading: true,
+                } as unknown as ReturnType<
+                    typeof useGetUserQueryHook.useGetUserQuery
+                >)
+        );
+        jest.spyOn(AuthenticatedHooks, 'default').mockImplementation(() => ({
+            isLoggedIn: false,
+            isLoading: false,
+            userData: {
+                userName: 'Pratiyush',
+                firstName: 'Pratiyush',
+                profilePicture: '',
+            },
+        }));
+        const { getByText } = renderWithRouter(
+            <Provider store={store()}>
+                <StandUp />
+            </Provider>,
+            {
+                query: { dev: 'true' },
+            }
+        );
+        expect(getByText('You are not Authorized')).toBeInTheDocument();
+    });
+
+    test('should not be able to render standup component if user is logged-in and component is not loading', () => {
+        jest.spyOn(useGetUserQueryHook, 'useGetUserQuery').mockImplementation(
+            (): ReturnType<typeof useGetUserQueryHook.useGetUserQuery> =>
+                ({
+                    isLoading: false,
+                } as unknown as ReturnType<
+                    typeof useGetUserQueryHook.useGetUserQuery
+                >)
+        );
+        jest.spyOn(AuthenticatedHooks, 'default').mockImplementation(() => ({
+            isLoggedIn: true,
+            isLoading: false,
+            userData: {
+                userName: 'Pratiyush',
+                firstName: 'Pratiyush',
+                profilePicture: '',
+            },
+        }));
+        const { getByText } = renderWithRouter(
+            <Provider store={store()}>
+                <StandUp />
+            </Provider>,
+            {
+                query: { dev: 'true' },
+            }
+        );
+        expect(getByText('Standup')).toBeInTheDocument();
+    });
+
+    test('should not be able to render standup component if user is authenticated and not loggedin', () => {
```

Comment:
how an someone be authenticated and not logged in?

Reply:
fixed